sources/Java.java

```diff
@@ -0,0 +1,41 @@
+import java.util.*;
+import java.io.*;
+import java.text.DecimalFormat;
+public class Java {
+
+	public static void main(String[] args) {
+		int range = Integer.parseInt(args[0]);
+		int trails = Integer.parseInt(args[1]);		
+		prng(range, trails);
+	}
+	static void prng(int numbers, int trails) {
+		int frequency[] = new int[numbers];
+		double probability[] = new double[numbers];
+		int start_time = (int) System.currentTimeMillis();
```

Comment:
Why are we avoiding using the native `long`?  Generally, I'd like to remove all type casting.